execution_workflows/DaPars/bin/convert_output.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+import sys
+import argparse
+import pandas as pd
+
+def parse_args(args=None):
+    Description = "Reformat DePars output file into the output files of differential challenge"
+    Epilog = "Example usage: python convert_to_tsv.py <FILE_IN> <IDENTIFICATION_OUT> <DIFFERENTIAL_OUT>"
+
+    parser = argparse.ArgumentParser(description=Description, epilog=Epilog)
+    parser.add_argument("FILE_IN", help="Input deAPA output txt file.")
+    parser.add_argument("IDENTIFICATION_OUT", help="Name of output file for identification challenge")
```

Comment:
See comment in README RE using distal coordinate of the loci column as the distal poly(A) site. DaPars uses a two site model, so if it predicts a proximal poly(A) site within the last exon then we should also report the distal site.

Reply:
Sounds good, so just to clarify, for each row in the output file, we want to obtain two things:
1. The predicted proximal APA site
2. The left end of the loci if orientation is negative, or the right end of the loci if orientation is positive?
If this is right, we'll then get twice the number of rows since each row in dapars' output file would generate two rows in our challenge's output file--one for proximal site and one for distal site. Is this the correct understanding?
@SamBryce-Smith